services/fluxmonitor.go

```diff
@@ -164,8 +164,10 @@ func (fm *FluxMonitor) Stop() {
 
 func (fm *FluxMonitor) canSubmitUpdated() {
 	if fm.state.CanSubmit {
+		logger.Info("Starting tickers because node is eligible to submit")
 		fm.startTickers()
 	} else {
+		logger.Info("Stoping tickers because node is not eligible to submit")
```

Comment:
Can we move these log lines into their respective functions, so they're only called once the tickers are actually started/stopped?

Reply:
Should also use the FM logger.